tier/repository.yaml

```diff
@@ -0,0 +1,12 @@
+name: tier
+description: A single tool to configure, orchestrate and manage your entire billing stack
+category: PRODUCTIVITY
+private: true
```

Comment:
you might need to mark this as public for now if you want to test it on one of your own clusters